watchtower/wtclient/backup_task_internal_test.go

```diff
@@ -192,9 +193,12 @@ func genTaskTest(
 			Index: index,
 		}
 
-		witnessType := input.CommitmentNoDelay
-		if tweakless {
+		var witnessType input.WitnessType
+		switch {
+		case chanType.IsTweakless():
 			witnessType = input.CommitSpendNoDelayTweakless
+		default:
+			witnessType = input.CommitmentNoDelay
```

Comment:
Still getting through the PR, but leaving a note here that this section will need to be updated to account for the new P2WSH output due to the 1 conf requirement for anchors (so our output in the breached transaction).

Reply:
this is done in a later commit

Reply:
done in a later commit!